app/controllers/abstract_xml_controller.rb

```diff
@@ -7,7 +7,11 @@ module AbstractXmlController
   # belongs here.
   ##############################################
 
+  #constant - words to ignore when autolinking
+  STOPWORDS = ["Mrs", "Mrs.", "Mr.", "Mr", "Dr.", "Dr"]
```

Comment:
We need to add "Miss" -- it shows up all over 

We also need to add "he", "she", and "it"